From akpm@osdl.org Tue Apr  4 00:19:26 2006
Message-Id: <200604040718.k347IxMq018140@shell0.pdx.osdl.net>
From: Andrew Morton <akpm@osdl.org>
Subject: PCI: 64 bit resources drivers pci changes sparc32 fix
To: akpm@osdl.org, greg@kroah.com, vgoyal@in.ibm.com, mm-commits@vger.kernel.org
Date: Tue, 04 Apr 2006 00:18:09 -0700


From: Andrew Morton <akpm@osdl.org>

arch/sparc/kernel/pcic.c:863: error: conflicting types for `pcibios_align_resource'
include/linux/pci.h:407: error: previous declaration of `pcibios_align_resource'

Cc: Vivek Goyal <vgoyal@in.ibm.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 arch/sparc/kernel/pcic.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/arch/sparc/kernel/pcic.c
+++ gregkh-2.6/arch/sparc/kernel/pcic.c
@@ -859,7 +859,7 @@ char * __init pcibios_setup(char *str)
 }
 
 void pcibios_align_resource(void *data, struct resource *res,
-			    unsigned long size, unsigned long align)
+			    u64 size, u64 align)
 {
 }
 
